pkg/component/openvpnclient.go

```diff
@@ -78,6 +80,100 @@ func (o OpenVPNClient) getFileMappings() [][]string {
 	}
 }
 
+type RevocationResponse struct {
+	Revoked    bool
+	RevokeTime time.Time
+}
+
+type ListOutput struct {
+	User       string
+	Valid_from string
+	Valid_to   string
```

Comment:
Same as above, `Valid_to -> ValidTo`.

Reply:
mmmm in go the standard is snake_case, not camel case btw. Anyway, if this is a super strict requirement, i will change it  :)